src/core/server/server.api.md

```diff
@@ -246,6 +248,15 @@ export interface HttpServiceStart {
     isListening: (port: number) => boolean;
 }
 
+// @public
+export interface IKibanaSocket {
+    // (undocumented)
+    getPeerCertificate(detailed: true): DetailedPeerCertificate | null;
+    // (undocumented)

```

Comment:
I'd expect api-extractor to merge all overloads in the one file.

Reply:
Yeah that's kinda strange. You might be able to use the `@inheritDoc` directive?

Reply:
nope, neither `@inheritDoc` nor `@link`. 
> The @link reference could not be resolved: The reference is ambiguous because "getPeerCertificate" has more than one declaration; you need to add a TSDoc member reference selector^M

seems the problem is not solved yet https://github.com/microsoft/web-build-tools/issues/881